src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
Yeah. I mean, if that makes any sense at all, not sure what else could go into the header, but if there's no reason to limit it to a Title then this would be the way you'd do that.